modules/06 Angular/Assignments/15/answer/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<app-header></app-header >
+<div id="container" [className]=" (bigPicSrc!=='')? 'blur' : '' ">
```

Comment:
What if I want to add one more class? :slightly_smiling_face:

Reply:
Thanks for your code review, if you like to add more then one css classes by some conditions you can use ngClass
[ngClass]="{
  'blur': bigPicSrc!=='',
'othet_css_class', other_condition
}"